docs/getting-started/quick-start.md

```diff
@@ -2,14 +2,195 @@
+
+
+In this quick start, we are going to leverage Service Binding as a way to easily and safely connect the application to the database service. 
+In order to do that, we'll need to create a Service Binding ressource which will trigger the Service Binding Operator to inject the binding metadatas into the application.
+
+
+1. Create the ServiceBinding custom resource to inject the bindings:
+
+```yaml
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+    name: spring-petclinic-rest
+    namespace: my-postgresql
+spec:
+    services:
+    - group: "crunchydata.com"
+      version: v1
+      kind: Pgcluster
+      name: hippo
+    - group: ""
+      version: v1
+      kind: Secret
+      name: hippo-hippo-secret
+    application:
+      name: spring-petclinic-rest
+      group: apps
+      version: v1
+      resource: deployments
+    mappings:
+    - name: type
+      value: "postgresql"
+```
+
+For simplicity, you can copy/paste the following command to create the resource:
+
+```bash
+kubectl apply -f https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/11e790fb1d23aa4d3ee03c260169a08b36fb25bc/service-binding.yaml
+```
+
+To learn more about creating service bindings, you can find more information on the following [document](../creating-service-bindings/creating-service-binding)..
+
+**We need to explain what's going on in the application:**
+- What has been injected?
+- Where it has been injected?
+- How the application found the metadata
+
+
+2. Let's now check how the application is behaving and setup the port forwarding of the application port to access it from our local environment
+
+```
+kubectl port-forward --address 0.0.0.0 svc/spring-petclinic-rest 9966:80 -n my-postgresql
+```
+
+3. Open [http://localhost:9966/petclinic](http://localhost:9966/petclinic), you should see a [Swagger UI][swagger] where you can play with the API.
+
+If you try to access list of all pets, you can see the application is now connected to the database and see the sample data initially configured:
+
+```
+$ curl -X GET "http://localhost:9966/petclinic/api/pets" -H "accept: application/json"
+[{"id":1,"name":"Leo","birthDate":"2000/09/07","type":{"id":1,"name":"cat"},
+"owner":{"id":1,"firstName":"George","lastName":"Franklin","address":"110...
+```
+
+## Next Steps
+
+In this sample, we setup a database and connected it to an application using the Service Binding operator to collect the connection metadata and expose them to the application.
+
+By using service bindings, developers are able to more easily leverage the services available to them on a Kubernetes cluster. 
+This method provides consistency accross different services and is repeatable for the developers. By remove the usual manual and error prone configuration, they benefit from a unified way to do application-to-service linkage.
+
+You can continue to learn more about Service Binding by:
+- link 1
```

Comment:
We need to provide links to the other content we are creating.

Reply:
Done. 1c27e60b4ef80faf802e6913a35b59dcc746f62a